content/theme/hugo-connectome.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Hugo Connectome Theme"
+github: https://github.com/kausalflow/connectome
+demo: https://hugo-connectome.kausalflow.com # Please make sure this links to the theme demo and not your personal/business site
```

Comment:
Please remove all the comments from the front-matter

Reply:
Thank you, @JugglerX .
I've removed the comments in commit `bb9a1ed`. 😄